src/controllers/user.controller.js

```diff
@@ -0,0 +1,177 @@
+import BaseRepository from '../repository/base.repository';
+import responseGenerator from '../helpers/responseGenerator';
+import utility from '../helpers/utils';
+
+import db from '../database/models';
+
+const { hashPassword, jwtSigner, verifyPassword } = utility;
+
+/**
+ * @class UserController
+ */
+class UserController {
+  /**
+   * @param {object} req - express request object
+   * @param {object} res - express response object
+   * @returns {object}- return user's account information
+   * @memberof UserController
+   */
+  static async createAccount(req, res) {
+    try {
+      const { username, email, password } = req.body;
+
+      const user = await BaseRepository.findOneByField(db.User, { email });
+
+      if (user) {
+        if (user.status === 'unverified') {
+          // TODO : SEND EMAIL TO USER
+          return responseGenerator.sendError(
+            res,
+            400,
+            `You already have an account. Kindly check your email address for a verification link.`
+          );
+        }
+
+        return responseGenerator.sendError(
+          res,
+          400,
+          'User with this email address already exist'
+        );
+      }
+
+      // create user
+      const encryptPass = hashPassword(password);
```

Comment:
complete the variable name. ` encryptPass ` -> ` encryptPassword`

Reply:
Thanks for the feedback. I will resolve the outlined comments.